airflow/www/static/js/tree.js

```diff
@@ -73,7 +73,7 @@ document.addEventListener('DOMContentLoaded', () => {
   treeDepth += 1;
 
   const innerWidth = window.innerWidth > 1200 ? 1200 : window.innerWidth;
```

Comment:
```suggestion
```
This line is no longer necessary then.

Reply:
@bbovenzi Thanks, I should have checked that upfront.